### DISCUSSION TITLE
Depth sum of organic matter weight

### DISCUSSION THREAD

**Discussion by kevinwolz** [ncss-tech/soilDB#337] [Q&A] [answered]
The [Soil Properties Map](https://casoilresource.lawr.ucdavis.edu/soil-properties/) shows "Soil Organic Matter" in kg/m3 as the "depth sum of all horizons". SSURGO has the `om_*` variables which are OM percentages, but I cannot find a variable for bulk density with which to calculate actual weights. Am I missing something? The Soil Properties Map also maps bulk density, so it must be in there somewhere!

**Comment by dylanbeaudette** [comment]
This calculation uses the Db at 1/3 bar water retention (`dbthirdbar_r` ) column from the `chorizon` table.

**Reply by kevinwolz** [reply]
Ah, so "db" is bulk density? That's what I get for just searching for "bulk density" in the metadata!

So that map is summing `om_r / 100 * dbthirdbar_r * (hzdepb_r - hzdept_r) * 10` across horizons?

**Comment by dylanbeaudette** [comment, marked as answer]
Yes, in the soil science and especially soil physics literature bulk density is represented as $D_{b}$. Hard to use subscripts in a database, so it is abbreviated as "db" or "Db". I've asked that the word "bulk density" be added to the SSURGO metadata, it is kind of confusing.

The calculation includes an additional term, "soil fraction", which is (1 - coarse fragment volume fraction.) Same as the calculation of SOC stock:

![soc-calculation-example-with-values](https://github.com/ncss-tech/soilDB/assets/624277/72c01b9e-66a7-4e75-9a9a-7e13b2e63f88)

**Reply by kevinwolz** [reply]
Ah yes. Here in the mollisols of Illinois I always forget about those things called rocks...

Thanks!
